tests/python/frontend/pytorch/test_forward.py

```diff
@@ -578,9 +578,15 @@ class Squeeze2(Module):
         def forward(self, *args):
             return args[0].squeeze(1)
 
+    class Squeeze3(Module):
+        def forward(self, *args):
+            return args[0].squeeze((1, 3))
```

Comment:
Probably this code can be modified and all these classes can be squeezed into one class. This one class will take the arguments for squeeze.

Reply:
I took a look in another tests. And many of them are implemented in the same style. So, let's keep it as you have implemented. Thank you!